client/src/components/Ticketing/ticketingmanager/Season/SeasonMain.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Udash_nav from '../udash_navbar';
+import InstancesPage from '../showings/InstancesPage';
+
+/**
+ * Main page to host seasons page
+ *
+ * @module
+ * @returns Seasons Page
+ */
+
+const SeasonsMain = () => {
+  return (
+    <div className='flex flex-row'>
```

Comment:
I know this is really tiny, but I think `flex-row` is the default flex-direction so its inclusion is redundant.

Reply:
Fixed!